ion_hash_test.go

```diff
@@ -166,7 +166,7 @@ func ionHashDataSource(t *testing.T) []testObject {
 								identityDigestList = append(identityDigestList, digestBytes)
 							case "final_digest":
 								digestBytes := readSexpAndAppendToList(t, reader)
-								identityFinalDigestList = append(identityFinalDigestList, digestBytes)
+								identityFinalDigestList = digestBytes
```

Comment:
Why bother with digestBytes? Why not just `identityFinalDigestList = readSexpAndAppendToList(t, reader)` ?

Reply:
Removed